canonical/lxd-ui
# Use remote image index

## Done

- Reference remote image indexes which are always up to date
- Drop local json dumps of image indexes (The cloud-images.ubuntu.com server introduced CORS headers, so we can load the data directly and don't need to keep a local copy)
- adding json dump for https://uk.lxd.images.canonical.com index file, as that server does not contain the CORS headers yet. But the https://images.linuxcontainers.org server will stop lxd support on 15th of January.

Fixes: WD-8211

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create a new instance and ensure images from the two cloud-images.ubuntu.com sources are available in the browse images modal: 1) Ubuntu minimal 2) Ubuntu
